docs/source/algorithms/Abins-v1.rst

```diff
@@ -33,6 +33,9 @@ theoretical spectrum for the given atom which is a sum over all considered quant
 also produce a total spectrum for the whole considered system. Dynamical structure factor S is calculated for
 all atoms in the system. If needed  a user can also include in a simulation elevated temperature.
 
+A description about the implemented working equations can be found :ref:`here <concepts contents>`
```

Comment:
This should link directly to the page. For now, I could not find `TheoreticalS` in the concepts page. I will try a clean build for that. I would rename that page anyway